msg/airflow_slip.msg

```diff
@@ -0,0 +1,3 @@
+uint64 timestamp		# time since system start (microseconds)
+float32 slip_rad # sideslip angle in radians
+bool valid # true if measurement is within sensor/calibration range, false otherwise
```

Comment:
If the `airflow_slip` and `airflow_aoa` have identical data/information, can we share a single uORB message?